src/java/utils/JsonHelper.java

```diff
@@ -14,4 +17,39 @@ public static JSONObject buildNoneLanguageMap(String label)
         languageMap.put("none", noneMap);
         return languageMap;
     }
+
+    /**
+    *  Make a hashmap with old @type from v2 API being keys and new type and format fields being values
+     *  to use in buildAnnotationResource
+    * */
+    private static HashMap buildBodyMap()
+    {
+        HashMap<String, JSONObject> map = new HashMap();
+        
+        JSONObject textMap = new JSONObject();
+        textMap.put("type", "TextualBody");
+        textMap.put("format", "text/plain");
+        map.put("cnt:ContentAsText", textMap);
+	
+        return map;
+
+    }
+
+    /**
+     * Make a body property for annotations according to IIIF v3 API using @type and value from v2
+     * works based on map build by buildResourceMap() and throws NoSuchElementException if type was not found among keys
+     * */
+    public static JSONObject buildAnnotationBody(String type, String value) throws NoSuchElementException
+    {
+        // since jsonobject extends object, casting should be safe
+        JSONObject body = (JSONObject) buildBodyMap().get(type);
+        if (body == null)
+        {
+            throw new NoSuchElementException(type + " was not found among supported types: " + buildBodyMap().keySet().toString());
+        }
+        body.put("language", "none");
```

Comment:
This is a little tricky. Web Annotation says this is fine, but IIIF wants it to always be an array https://iiif.io/api/presentation/3.0/#language 

Generally, in IIIF if it might be an array, they prefer that it is always one, but the individual cases are all laid out in the API.